platforms/hyperledger-fabric/charts/fabric-peernode/templates/node-statefulset.yaml

```diff
@@ -135,23 +135,23 @@ spec:
 {{- else }}
           function getPeerTlsSecret {
             KEY=$1
-            kubectl get secret ${KEY} --namespace {{ .Release.Namespace }} --output="jsonpath={.data.cacrt}" | base64 -d > ${OUTPUT_PATH}/ca.crt
-            kubectl get secret ${KEY} --namespace {{ .Release.Namespace }} --output="jsonpath={.data.servercrt}" | base64 -d > ${OUTPUT_PATH}/server.crt
-            kubectl get secret ${KEY} --namespace {{ .Release.Namespace }} --output="jsonpath={.data.serverkey}" | base64 -d > ${OUTPUT_PATH}/server.key
+            kubectl get secret ${KEY} --namespace {{ .Release.Namespace }} --output="jsonpath={.data.cacrt}" | base64 -d >> ${OUTPUT_PATH}/ca.crt
+            kubectl get secret ${KEY} --namespace {{ .Release.Namespace }} --output="jsonpath={.data.servercrt}" | base64 -d >> ${OUTPUT_PATH}/server.crt
+            kubectl get secret ${KEY} --namespace {{ .Release.Namespace }} --output="jsonpath={.data.serverkey}" | base64 -d >> ${OUTPUT_PATH}/server.key
           }
 
           function getPeerMspSecret {
             KEY=$1
-            kubectl get secret ${KEY} --namespace {{ .Release.Namespace }} --output="jsonpath={.data.admincerts}" | base64 -d > ${OUTPUT_PATH}/admincerts/admin.crt
```

Comment:
does the container have jq? its better to run the get secret command only one time to get the whole data object and the use jq to get individual certs from it

Reply:
Done.
Also made the same change in the charts: fabric-ca-server, fabric-orderernode, fabric-genesis and fabric-cli